src/test/run-make-fulldeps/print-rustc-path/Makefile

```
@@ -0,0 +1,6 @@
+-include ../tools.mk
+
+-include ../tools.mk
+
+all:
+	$(RUSTC) --print cfg | $(CGREP) $(RUSTC)
```

```suggestion
	$(RUSTC) --print rustc-path | $(CGREP) $(RUSTC)
```

and maybe also test a combination with other print args?

---

🤦stupid copy-paste error